tests/basic_test.c

```diff
@@ -496,7 +496,7 @@ int fuzi_q_set_test_server_ctx(fuzi_q_test_config_t* test_config, fuzi_q_ctx_t*
         }
     }
 
-    fuzi_q_ctx->quic = picoquic_create_and_configure(&config, picoquic_demo_server_callback, NULL, test_config->simulated_time,
```

Comment:
See previous comments.

```suggestion
fuzi_q_ctx->quic = picoquic_create_and_configure(&config, NULL, NULL, test_config->simulated_time
```